### PR TITLE
Postgres exporter rename

### PR DESCRIPTION
**Pull request by mseidl**
The postgres exporter package has been renamed.  

This PR has to be timed with a PR from spacewalk, uyuni and the packages in obs.

### CODE REVIEW

File: prometheus-exporters-formula/prometheus-exporters/map.jinja [hunk 1]
```diff
@@ -34,7 +34,7 @@
         'apache_exporter_package': 'golang-github-lusitaniae-apache_exporter',
         'apache_exporter_service': 'prometheus-apache_exporter',
         'apache_exporter_service_config': '/etc/sysconfig/prometheus-apache_exporter',
-        'postgres_exporter_package': 'golang-github-wrouesnel-postgres_exporter',
+        'postgres_exporter_package': 'prometheus-postgres_exporter',
```
Comment (witekest) [inline prometheus-exporters-formula/prometheus-exporters/map.jinja:37]: Have you checked if the package builds for RedHat?

**Review by witekest — COMMENTED** [review]
Looks good in general.
I wasn't aware that we update the RedHat package as well. Have you tested if it builds?

You'll also have to update [susemanager-sls](https://github.com/uyuni-project/uyuni/blob/master/susemanager-utils/susemanager-sls/salt/srvmonitoring/enable.sls#L14).

We still have to wait with releasing it until the exporter package actually lands in client tools repositories.

Please also think about removing the old package afterwards.

**Comment by witekest** [comment]
> Changelog needs changes. Don't you need to change the SPEC as well?

Changelog formatting updated.
Version number bumped in the spec file.

**Review by juliogonzalez — APPROVED** [review]
LGTM

**Pull request merged**